scilpy/tracking/propagator.py

```diff
@@ -275,7 +278,8 @@ def __init__(self, datavolume, step_size, rk_order, dipy_sphere,
         """
         super().__init__(datavolume, step_size, rk_order, space, origin)
 
-        self.sphere = dipy.data.get_sphere(dipy_sphere)
+        self.sphere = HemiSphere.from_sphere(
```

Comment:
This is breaking no? Based on method `get_sphere_neighbour`, a whole sphere is required for tracking (there is no absolute value on the `cos`). Also, using a whole sphere enables tracking on asymmetric ODFs.

 https://github.com/scilus/scilpy/blob/1b49e37ed7c419db20a32d4f17087d357e737d03/scilpy/reconst/utils.py#L39